src/Setup/InstallSchema.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EasyTranslate\CompatMageplazaBlog\Setup;
+
+use Magento\Framework\DB\Ddl\Table;
+use Magento\Framework\Setup\InstallSchemaInterface;
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\Framework\Setup\SchemaSetupInterface;
+use Zend_Db_Exception;
+
+class InstallSchema implements InstallSchemaInterface
+{
+    /**
+     *
```

Comment:
Please add an explanation why we use install schema and not DB schema with a reference to the Magento core issue.